lib/file-source/Cargo.toml

```diff
@@ -20,7 +20,7 @@ tokio = { version = "0.2.13", features = ["rt-core", "blocking", "time"] }
 serde = { version = "1.0.117", features = ["derive"] }
 serde_json = "1.0.33"
 chrono = { version = "0.4.19", features = ["serde"] }
-dashmap = "3.11.10"
+dashmap = "4.0.0"
```

Comment:
Um, is this branched off of master? `lib/file-source` is already at `dashmap = "4.0.0"`

Reply:
Oh, DUH, commented on the wrong issue.